pyzeebe/channel/insecure_channel.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Optional
+
+import grpc
+
+from pyzeebe.channel.channel_options import get_channel_options
+from pyzeebe.channel.utils import create_address
+
+
+def create_insecure_channel(
+    hostname: Optional[str] = None,
+    port: Optional[int] = None,
+    channel_options: Optional[Dict] = None
+) -> grpc.aio.Channel:
```

Comment:
Consider simply `create_channel(secure=False)`? Not sure what the advantage is with separated `create_secure_channel`/`create_insecure_channel`

Reply:
You can provide your own `grpc.ChannelCredentials` when using `create_secure_channel`. This way we avoid confusion where users may try:

```python
create_secure_channel(secure=False, channel_credentials=some_credendtials)
```